component/main.jsonnet

```diff
@@ -7,4 +7,5 @@ local params = inv.parameters.nfs_client_provisioner;
 
 // Define outputs below
 {
+  '00_namespace': kube.Namespace(params.namespace),
```

Comment:
And now I would add the StorageClass here, as in https://github.com/projectsyn/component-csi-cloudscale/blob/master/component/main.jsonnet

Reply:
Please improve the documentation for component-storageclass, this is highly non-obvious at the moment